interface/wx/dc.h

```diff
@@ -1842,6 +1842,113 @@ class wxDCTextColourChanger
 };
 
 
+/**
+    @class wxDCTextBgColourChanger
+
+    wxDCTextBgColourChanger is a small helper class for setting a background
+    text colour on a wxDC and unsetting it automatically in the destructor,
+    restoring the previous one.
+
+    @library{wxcore}
+    @category{gdi}
+
+    @see wxDC::SetTextBackground(), wxDCFontChanger, wxDCPenChanger, wxDCBrushChanger,
```

Comment:
Perhaps references to`wxDCTextBgModeChanger`  and `wxDCTextColourChanger` could be added here as well (and perhaps references to classes not related to drawing text - `wxDCPenChanger`, `wxDCBrushChanger`, `wxDCClipper` - removed if you feel there is too many references)?